spec/paperclip/attachment_spec.rb

```diff
@@ -565,16 +676,16 @@ def thumb; "-thumb"; end
 
   context "An attachment with conditional :styles that is a proc" do
     before do
-      rebuild_model styles: lambda{ |attachment| attachment.instance.other == 'a' ? {thumb: "50x50#"} : {large: "400x400"} }
+      rebuild_model styles: lambda{ |attachment| attachment.instance.other == "a" ? {thumb: "50x50#"} : {large: "400x400"} }
```

Comment:
Line is too long. [124/80]<br>Space missing to the left of {.<br>Space inside { missing.<br>Space inside } missing.